activerecord/lib/active_record/connection_adapters/abstract/schema_creation.rb

```diff
@@ -93,6 +101,10 @@ def options_include_default?(options)
             options.include?(:default) && !(options[:null] == false && options[:default].nil?)
           end
 
+          def index_in_create(table_name, column_name, options)
```

Comment:
I think defining `index_in_create` as an interface of abstract adapter is not good. `index_in_create` is only defined on `abstract_mysql_adapter.rb` so I feel it is not an interface.